pkg/controller/metrics/metrics.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package metrics provides metrics declaration for the antrea controller.
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	OpsAppliedToGroupProcessed = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
Prometheus stats are prefixed with some identifier of their origin.
e.g "go_threads" or "felix_active_local_endpoints".

Consider changing into antrea_applied_to_group_processed or antrea_ctl_ applied_to_group_processed if we want to distinguish controller metrics from agents'.
I think that the latter makes sense.

Reply:
Thanks, I have updated them to `antrea_controller_*`.